src/cli.rs

```diff
@@ -1645,4 +1675,48 @@ mod tests {
         assert_eq!(parse_encoding("notreal").is_err(), true);
         assert_eq!(parse_encoding("").is_err(), true);
     }
+
+    #[test]
+    fn parse_format_options() {
+        let invalid_format_options = vec![
+            // malformed strings
+            ":8",
+            "json.indent:",
+            ":",
+            "",
+            "json.format:true, json.indent:4",
+            // invalid values
+            "json.indent:-8",
+            "json.format:False",
```

Comment:
HTTPie actually supports this case. It seems to be case-insensitive in general, even for the keys:

```py
            path, value = option.lower().split(':')
```
https://github.com/httpie/cli/blob/e52a60e67ca15291c9ab18f395d4d7af3a6204ff/httpie/cli/argtypes.py#L220

(so `JSON.format:TRUE` is OK.)